packages/web-pkg/src/components/Spaces/SpaceImageModal.vue

```diff
@@ -132,10 +134,15 @@ onMounted(async () => {
 })
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
 .space-image-modal {
   &-image-container {
     max-height: 400px;
   }
+
+  .cropper-line,
+  .cropper-point {
+    background-color: var(--oc-role-secondary-container) !important;
```

Comment:
What about `oc-role-outline` instead? Sounds more appropriate to me, semantically.
Also: Do you want to apply the same color role to the outline of `cropper-view-box`? That has an `outline: 1px solid #39f;` by default and that's the same blue like the one you were overwriting with the cropper-line and cropper-point.